ansible/roles/pg_backup/defaults/main.yml

```diff
@@ -0,0 +1,18 @@
+postgresql_version: '9.4'
+postgresql_port: 5432
+postgresql_backup_dir: "{{ encrypted_root }}/backups/postgresql"
+postgresql_dir_original_path: /var/lib/postgresql
+
+nadir_hour: 0
+postgresql_backup_days: 3
+postgresql_backup_weeks: 4
+backup_postgres: plain
+postgres_s3: false
+hot_standby_server: false
```

Comment:
What's the significance of this in this role?

Reply:
pgbackrest uses this variable a couple of times, this is to replace using |default(false) repeatedly